Feature/bing 61 design search dropdown

palianskas opened the pull request:
Added dropdown to appbar-search to display results (mocked at the moment). On dropdown item click a _title-dialog_  is opened with the data from the dropdown item's title

palianskas committed:
BING-61: design search-dropdown prototype

palianskas committed:
BING-61: move appbar search to separate component

palianskas committed:
BING-61: fix searchBar and dropdown misalignment

palianskas committed:
BING-61: fix popover shifting focus off of searchBar

palianskas committed:
BING-61: adjust dropdown animation origin and param names

palianskas committed:
BING-61: design dropdown item

palianskas committed:
BING-61: clean up search component

palianskas committed:
BING-61: change dropdown base to custom component implementation

palianskas committed:
BING-61: fix dropdown close jumpy transition

palianskas committed:
BING-61: adjust transition timings

palianskas committed:
BING-61: update title object's props used by title-dialog

palianskas committed:
BING-61: create 'open title-dialog on dropdown item click' functionality

palianskas committed:
BING-61: add onhover hint to dropdown item's and title-dialog's titles

MatasRut reviewed (commented):
CRLF line ending errors appear on every file changed. Reformat files and make sure git is not converting line endings

MatasRut reviewed (changes requested):
Search dropdown items should have a hover effect and the mouse cursor should change to the clicking hand. Right now they do not look clickable.

palianskas committed:
BING-61: fix dropdown item onclick not firing

palianskas commented:
> CRLF line ending errors appear on every file changed. Reformat files and make sure git is not converting line endings

Seems like a false flag. Marking as resolved

> Search dropdown items should have a hover effect and the mouse cursor should change to the clicking hand. Right now they do not look clickable.

Moved dropdown's `handleClose` to a different callback, fixed in latest commit

palianskas committed:
BING-61: fix dropdown not closing after off-click

palianskas committed:
BING-61: add pointer cursor to dropdown item's onHover

palianskas commented on app/bingey-ui/src/components/Watchlist/TitleDialog/titleDialogStyle.scss:
```diff
@@ -1,22 +1,28 @@
-.title{
-    margin-right: auto !important;
+.title {
```
Add overflow ellipsis to long title names

palianskas committed:
BING-61: add hover transition to dropdown items

palianskas committed:
Merge branch 'development' of github.com:palianskas/bingey into feature/BING-61-design-search-dropdown

palianskas committed:
BING-61: resolve merge conflicts

palianskas committed:
BING-62: minor fixes
